packages/plugin-ext/src/main/browser/plugin-frontend-contribution.ts

```diff
@@ -54,15 +68,174 @@ export class PluginApiFrontendContribution implements CommandContribution {
             execute: () => this.pluginExtDeployCommandService.deploy()
         });
 
-        // this command only for compatibility reason
-        commands.registerCommand({ id: 'workbench.action.closeActiveEditor' }, {
-            execute: () => commands.executeCommand('core.close.tab')
-        });
-
         commands.registerCommand(OpenUriCommandHandler.COMMAND_METADATA, {
             execute: (arg: URI) => this.openUriCommandHandler.execute(arg),
             isVisible: () => false
         });
+
+        // https://code.visualstudio.com/docs/getstarted/keybindings#_navigation
+        /*
+         * internally, in VS Code, any widget opened in the main area is represented as an editor
+         * operations below apply to them, but not to side-bar widgets, like the explorer
+         *
+         * in Theia, there are not such difference and any widget can be put in any area
+         * because of it we filter out editors from views based on `NavigatableWidget.is`
+         * and apply actions only to them
+         */
+        commands.registerCommand({ id: 'workbench.action.files.newUntitledFile' }, {
+            execute: () => open(this.openerService, createUntitledResource().uri)
+        });
+        commands.registerCommand({ id: 'workbench.action.files.openFile' }, {
+            execute: () => commands.executeCommand(WorkspaceCommands.OPEN_FILE.id)
+        });
+        commands.registerCommand({ id: 'workbench.action.files.openFolder' }, {
+            execute: () => commands.executeCommand(WorkspaceCommands.OPEN_FOLDER.id)
+        });
+        commands.registerCommand({ id: 'workbench.action.files.save', }, {
+            execute: (uri?: monaco.Uri) => {
+                if (uri) {
+                    const uriString = uri.toString();
+                    const widget = this.shell.widgets.find(w => {
+                        const resourceUri = Saveable.is(w) && NavigatableWidget.is(w) && w.getResourceUri();
+                        return (resourceUri && resourceUri.toString()) === uriString;
+                    });
+                    if (Saveable.is(widget)) {
+                        Saveable.save(widget);
+                    }
+                } else {
+                    this.shell.save();
+                }
+            }
+        });
+        commands.registerCommand({ id: 'workbench.action.files.saveAll', }, {
```

Comment:
seems to be duplicating 
https://github.com/theia-ide/theia/blob/5cbb4f19d65058f3875fb303ac0ee86a35140491/packages/core/src/browser/common-frontend-contribution.ts#L447-L448

Reply:
For now, yes, i can delegate it. It does not mean that at some point we will need add some custom logic to align with VS Code behaviour.